api/scripts/tcr/tcr.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+alteredFilePaths=$(git ls-files -m '*.js');
+files=$@
+
+function lint () {
+  npx eslint $alteredFilePaths --fix
+}
+
+function test () {
+  npm run test:api:path $files -- --bail
```

Comment:
Suite du changement de `$@` :
```suggestion
  npm run test:api:path "${files[@]}" -- --bail
```

Reply:
Done.